src/TikaOnDotnet.TextExtractor/Extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TikaOnDotNet.TextExtraction
+{
+    /// <summary>
+    /// Contains extension methods to standard .NET objects
+    /// </summary>
+	public static class Extensions
```

Comment:
Maybe we need an Extensions namespace/folder and the extension types can live there.